app/controllers/homepage_controller.rb

```diff
@@ -23,36 +24,28 @@ def dashboard
       goal_values = Array.new
       current_effective_user.workout.each do | workout |
         workout.task.each do |task|
+          puts "G unit name: "
+          puts g.unit.name
           if task.exercise_id == g.exercise_id then
             task.exercise_set.each do | exercise_set |
-              goal_values.push(Array.new([exercise_set.created_at.strftime("%D %H:%M"), exercise_set.rep_value]))
+              if exercise_set.rep_unit == g.unit.name then
```

Comment:
What does this mean?

Reply:
This section populates data to add to the dashboard graphs, so this prevents exercise logs from being added that don't apply for the goal units. So for now if a goal is Run "60 minutes" for example and someone logs Run "60 seconds" for an exercise, it doesn't populate the graph.

Reply:
Sweet, thanks for the clarity!